src/rsz/src/CMakeLists.txt

```diff
@@ -35,12 +35,22 @@
 
 include("openroad")
 
-swig_lib(NAME      rsz
```

Comment:
Here I decided to split out the binding into a library of its own because we are polluting it's include path a lot (see below in this file).